integration-test/test/basic.bats

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bats
+
+load test_helper
+
+@test "meta: docker is installed" {
+    run docker version
+    echo "$output">&2
+    [ "$status" -eq 0 ]
```

Comment:
@boumenot nope `$status` contains the exitcode of the command that is last executed by prefix `run ...` thanks to bats. I type output here because if it fails on the `[ "$status" -eq 0 ]` status line, we'll see the output of the command and tell why it failed.